pkg/verify/testdata/manifests/only-first-used.yaml

```diff
@@ -0,0 +1,112 @@
+# NOTE: This file is parsed by CVO's lib/Manifest ParseManifests method which
+# reverses the order of the below config map. The valid one is therefore the
+# last one in the below list.
```

Comment:
I don't think it reverses the order, see [this test-case][1].  One issue is that `wantVerifiers` is currently an integer.  I'd rather see `expectedVerifier string` and `expectedError string` (or `*regexp.Regexp` for either, if we need that flexibility), to be compared with the output of [`got.String()`][2] and `err.Error()` like [this][3].

[1]: https://github.com/openshift/cluster-version-operator/blob/16d3d8465b8e3f3c557becab28763d6cc18829a6/lib/manifest_test.go#L66-L99
[2]: https://github.com/openshift/cluster-version-operator/blob/16d3d8465b8e3f3c557becab28763d6cc18829a6/pkg/verify/verify.go#L82-L83
[3]: https://github.com/openshift/cluster-version-operator/blob/692a8a37b52c2bc00c45fb84622ba2d5ed3125bb/pkg/verify/store/serial/serial_test.go#L86-L98

Reply:
Yeah, it wasn't reversing order but instead my failure to separate the manifests with `---` causing my issue. Works as expected now.
I also made the changes to the test case per your recommendation.